### ISSUE TITLE
[Feature Request] If thoughts could be shared with a group of people for collaboration

### ISSUE THREAD

**Issue by harshitpthk** [n3xah/issues#15]
If thoughts could be shared (through a hyper link, or even as a printable summary of what and how things were discussed with the agent to showcase why you arrived at a conclusion)

**Comment by brentbahry** [comment]
Yeah I think there's a lot of room up for collaboration/sharing. For now, there's the minimal copy button that you can click (per thought) that copies the thought and all children as markdown. The link that's generated will only work for you though.
